includes/class-wc-pagarme-api.php

```diff
@@ -326,15 +326,15 @@ public function generate_transaction_data( $order, $posted ) {
 			if ( apply_filters( 'wc_pagarme_allow_credit_card_installments_validation', isset( $posted['pagarme_installments'] ), $order ) ) {
 				$_installment = $posted['pagarme_installments'];
 
+				$data['installments'] = $_installment;
```

Comment:
Acho que pode substituir essa linha então:

```php
$_installment = $posted['pagarme_installments'];
```

Isso deve ser sempre integer? Se for podemos já trocar por:

$data['installments'] = absint( $posted['pagarme_installments'] );

Reply:
a variável $_installment é usada logo abaixo, pra uma validação

Reply:
Tudo tranquilo.